integration_tests/src/main/python/udf_cudf_test.py

```diff
@@ -12,16 +12,21 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import pytest
 import pandas as pd
+import pytest
 import time
 from typing import Iterator
 from pyspark.sql import Window
 from pyspark.sql.functions import pandas_udf, PandasUDFType
+from spark_init_internal import spark_version
 from spark_session import with_cpu_session, with_gpu_session
 from marks import allow_non_gpu, cudf_udf
 
 
+pytestmark = pytest.mark.skipif(spark_version() >= '3.1.0',
```

Comment:
nit: strings are compared lexicographically in python, so `"3.10.0" > "3.2.0" => False`. This does not affect anything in our case but could be better to convert it to tuple first then do the compare

Reply:
Goog catch

Reply:
updated